lib/state_machines/audit_trail/backend.rb

```diff
@@ -1,4 +1,4 @@
-class StateMachines::AuditTrail::Backend < Struct.new(:transition_class, :owner_class, :context)
+class StateMachines::AuditTrail::Backend < Struct.new(:transition_class, :owner_class, :options)
```

Comment:
I thought it wise to combine `:context` and the new plymorphic option `:as` into an `options` Hash, rather than grow an ever-expanding arg list. My other branch has `:as` as a separate options arg, leaving `:context` alone; let me know if you would prefer it that way.